content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -15,35 +15,81 @@ The release of InfluxDB Enterprise 1.9 is different from previous InfluxDB Enter
 in that there is no corresponding InfluxDB OSS release.
 (InfluxDB 1.8.x will continue to receive maintenance updates.)
 
+#21015: build: upgrade to go 1.15.10
+
 ### Features
 - Support user-defined *node labels*.
   Node labels let you assign arbitrary key-value pairs to meta and data nodes in a cluster.
   For instance, an operator might want to label nodes with the availability zone in which they're located.
 - Improve performance of `SHOW SERIES CARDINALITY` and `SHOW SERIES CARDINALITY from <measurement>` InfluxQL queries.
   These queries now return a `cardinality estimation` column header where before they returned `count`.
-- Improve diagnostics for license problems. Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
+- Improve diagnostics for license problems.
+  Add [license expiration date](/enterprise_influxdb/v1.9/features/clustering-features/#entitlements) to `debug/vars` metrics.
 - Add improved [ingress metrics](/enterprise_influxdb/v1.9/administration/config-data-nodes/#ingress-metric-by-measurement-enabled--false) to track points written by measurement and by login.
-- Support authorization for Kapacitor via LDAP.
+- Support authentication for Kapacitor via LDAP.
 - Support for [configuring Flux query resource usage](/enterprise_influxdb/v1.9/administration/config-data-nodes/#flux-controller) (concurrency, memory, etc.).
 - Upgrade to [Flux v0.113.0](/influxdb/cloud/reference/release-notes/flux/#v01130-2021-04-21).
 
+
+
+
+- feat(prometheus): update prometheus remote protocol <!-- https://github.com/influxdata/influxdb/pull/17814 -->
+- feat: meancount aggregation for WindowAggregate pushdown in enterprise <!-- https://github.com/influxdata/influxdb/pull/21291 -->
+- feat(tsi): optimize series iteration <!-- https://github.com/influxdata/influxdb/pull/20544 -->
+- fix: group by returns multiple results per group in some circumstances <!-- https://github.com/influxdata/influxdb/pull/21631 -->
+- feat: add arguments to flux to function <!-- https://github.com/influxdata/influxdb/pull/20873 -->
+- feat: Add WITH KEY to show tag keys <!-- https://github.com/influxdata/influxdb/pull/20793 -->
```

Comment:
can you check to see that this appears in the InfluxQL spec too?  /enterprise_influxdb/v1.9/query_language/explore-schema/#show-tag-keys

you can borrow the WITH syntax from:
enterprise_influxdb/v1.9/query_language/explore-schema/#show-tag-values

example: SHOW TAG KEYS FROM cpu WITH KEY IN (host, region)